{"uuid":"ecc27f5c-5c8d-4865-b104-ab302c5adf4a","historyId":"1d9f8c912b7bc9bfcd5cc6cde388c477","fullName":"APi_Negative.Accounting.SG_NG_Cluster_Accounting.Transaction_Ledger","labels":[{"name":"package","value":"APi_Negative.Accounting.SG_NG_Cluster_Accounting"},{"name":"testClass","value":"APi_Negative.Accounting.SG_NG_Cluster_Accounting"},{"name":"testMethod","value":"Transaction_Ledger"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"untitled"},{"name":"subSuite","value":"APi_Negative.Accounting.SG_NG_Cluster_Accounting"},{"name":"host","value":"DESKTOP-L0TTVHB"},{"name":"thread","value":"20244@DESKTOP-L0TTVHB.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"the Transaction_Ledger"}],"links":[],"name":"Transaction_Ledger","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [400] but found [200]","trace":"java.lang.AssertionError: expected [400] but found [200]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1577)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat org.testng.Assert.assertEquals(Assert.java:665)\r\n\tat APi_Negative.Accounting.SG_NG_Cluster_Accounting.Transaction_Ledger(SG_NG_Cluster_Accounting.java:102)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg3","value":"400"},{"name":"arg2","value":"null"},{"name":"arg4","value":"endDate null"},{"name":"arg1","value":"\"2024-01-01\""},{"name":"arg0","value":"\"fdfb5f2c-43d0-4c9b-83c0-b703b5d99d3e\""}],"start":1750667568181,"stop":1750667568495}